packages/webpack.web.config.js

```diff
@@ -50,6 +50,13 @@ module.exports = (env, argv) => {
             new webpack.IgnorePlugin({
                 resourceRegExp: /ps-list/, // matches the path in the require() statement
             }),
+            /**
+             * HACK: the glob module breaks Web mode if imported, BUT we still dynamically import this module for non web mode
+             * environments. The following allows compilation to pass by never bundling the module in the final output for web mode.
+             */
+            new webpack.IgnorePlugin({
+                resourceRegExp: /glob/, // matches the path in the require() statement
```

Comment:
is this intended to match `global` or what CI failure motivated this?

in the "test web" CI I see this error:

```
2025-04-07T17:25:48.4129595Z Error: Cannot find module './globalState'
2025-04-07T17:25:48.4148372Z 	at webpackMissingModule (file:///home/runner/work/aws-toolkit-vscode/aws-toolkit-vscode/packages/amazonq/dist/test/web/testRunnerWebCore.js#vscode-extension:159675:92)
```